format/format.go

```diff
@@ -492,7 +492,17 @@ func (w *writer) writeHead(head *ast.Head, isDefault, isExpandedConst bool, o fm
 			w.write("]")
 		}
 	}
-	if head.Value != nil && (head.Key != nil || ast.Compare(head.Value, ast.BooleanTerm(true)) != 0 || isExpandedConst || isDefault) {
+
+	if head.Value != nil &&
+		(head.Key != nil || ast.Compare(head.Value, ast.BooleanTerm(true)) != 0 || isExpandedConst || isDefault) {
+
+		if head.Location == head.Value.Location && head.Name != "else" {
+			// If the value location is the same as the location of the head,
+			// we know that the value is generated, i.e. f(1)
+			// Don't print the value (` = true`) as it is implied.
```

Comment:
Bringing some Regal tricks here. If there's a more idiomatic way of checking this, do let me know.

Reply:
💭 `ast.Expr` has a `Generated` boolean, but I suppose that's of no use here, since the Value is a Term

Reply:
Indeed :( Oh well, at least the comment should help explain what's going on.